docs/lightning/rtl.md

```diff
@@ -1,5 +1,83 @@
 # Ride the lightning (RTL)
 
-GUI to manage lightning channels
+A GUI to manage lightning channels
 
-work in progress
+## Using Ride the Lightning (RTL) Wallet
+
+### Logging into RTL
+
+Ride the Lightning is a Lightning wallet and node management tool accessible via a web interface, and is built into myNode.
+RTL is a creation of Suheb/saubyk, and complete information can be found at [Github](https://github.com/ride-the-lightning/RTL).
+The developer can be contacted via Twitter at: @suhub__ or @RTL_App.
+
+You can use RTL from any browser that is able to access your myNode installation.
+
+If you are accessing RTL from inside the network on which your myNode installation resides, you can open a browser window and enter <your.mynode.network.IP:3010> into the address bar.
+
+If you are accessing RTL remotely using the premium features: VPN or Tor, you can open a browser window and enter either <your.home.IP.address:3010> or <your.Tor.hidden.service.address:3010>, depending on which premium feature you are using at the time.
+
+Premium users can get all of their Tor Hidden Service addresses by going to the myNode home page and clicking on the "Tor Services" button in the "Services" area of the page.
+
```

Comment:
we should move this to some other page (possibly Tor or remote access)

Reply:
Ok